src/main/java/org/squiddev/plethora/integration/chickens/MetaChickens.java

```diff
@@ -0,0 +1,101 @@
+package org.squiddev.plethora.integration.chickens;
+
+import com.google.common.collect.Maps;
+import com.setycz.chickens.ChickensMod;
+import com.setycz.chickens.config.ConfigHandler;
+import com.setycz.chickens.entity.EntityChickensChicken;
+import com.setycz.chickens.registry.ChickensRegistry;
+import com.setycz.chickens.registry.ChickensRegistryItem;
+import net.minecraft.entity.Entity;
+import net.minecraft.nbt.NBTTagCompound;
+import org.squiddev.plethora.api.Injects;
+import org.squiddev.plethora.api.converter.DynamicConverter;
+import org.squiddev.plethora.api.meta.BaseMetaProvider;
+import org.squiddev.plethora.api.meta.IMetaProvider;
+import org.squiddev.plethora.api.meta.NamespacedMetaProvider;
+import org.squiddev.plethora.api.method.IPartialContext;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Map;
+
+@Injects(ChickensMod.MODID)
+public final class MetaChickens {
+	private MetaChickens() {
+	}
+
+	@Nullable
+	public static final DynamicConverter<Entity, EntityChickensChicken> GET_CHICKEN_FROM_ENTITY =
```

Comment:
Sorry, just actually noticed this. You shouldn't need need converters to upcast things. Everything is done dynamically, so if something is an `EntityChickensChicken` (even if it's only typed as `Entity`) then we'll apply the appropriate methods/meta providers/converters.

Definitely leads to some unintended consequences at times, but generally it's useful.

Reply:
I'll have to double check that one; it's fairly likely that came up when I hadn't quite figured out the Entity Sensor and I was just trying "maybe _THIS_ is why it isn't doing the thing?"

Reply:
Yeah, that appears to be my lack of experience with the Entity Sensor; everything worked with that bit commented out, so it has been removed.